pkg/operator/ceph/controller/spec.go

```diff
@@ -798,3 +800,9 @@ func ConfigureExternalMetricsEndpoint(ctx *clusterd.Context, monitoringSpec ceph
 func extractMgrIP(rawActiveAddr string) string {
 	return strings.Split(rawActiveAddr, ":")[0]
 }
+
+func GetContainerImagePullPolicy(containerImagePullPolicy string, opConfig OperatorConfig) *v1.PullPolicy {
+	imagePullPolicy := k8sutil.GetValue(opConfig.Parameters, "ROOK_CEPH_IMAGE_PULL_POLICY", containerImagePullPolicy)
```

Comment:
If the pull policy is blank, let's default to `IfNotPresent`. The setting will be blank by default after upgrades.

Reply:
So no need to have `containerImagePullPolicy` as input param?

Reply:
no, once you get the value check if it is empty, if empty assign default otherwise continue. Something like
```go
imagePullPolicy := k8sutil.GetValue(opConfig.Parameters, "ROOK_CEPH_IMAGE_PULL_POLICY", containerImagePullPolicy)
if imagePullPolicy == ""{
      imagePullPolicy = "IfNotPresent"
}
```

Reply:
Ok got it, thanks!